explorer/scan/local_scanner.go

```diff
@@ -205,7 +205,7 @@ func CreateProgressBar(discoveredAssets *DiscoveredAssets, disableProgressBar bo
 	return multiprogress, nil
 }
 
-func (s *LocalScanner) distributeJob(job *Job, ctx context.Context, upstream *upstream.UpstreamConfig) (*explorer.ReportCollection, error) {
+func (s *LocalScanner) distributeJob(ctx context.Context, job *Job, upstream *upstream.UpstreamConfig) (*explorer.ReportCollection, error) {
```

Comment:
This was my editor telling me that `ctx` should always be the first argument of a func, it is an internal function so it shouldn't be a problem, happy to remove this if we don't like it here.